content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,277 @@
+
+* Target specific functionality within the Editor, using [EditorPythonBindings and EditorTest](/docs/user-guide/testing/parallel-pattern/).
+* Rely on external crash-handling (editor crash causes a test script crash).
+
+To integrate with the Editor interpreter, create a test which uses `EditorPythonBindings`. These tests **must not** be in a file starting with `test_` or `tests_`, to avoid accidentally registering as failing tests.
+
+{{< note >}}
+PyTest is not used within the Editor interpreter, and as a consequence PyTest functionality is unavailable to tests which run in the Editor interpreter. Avoid dependencies on PyTest fixtures when designing these tests.
+
+EditorTest still uses PyTest to manage tests, and additionally handles external crash monitoring, batching, and parallelism. If this tool does not meet your needs, please reach out with a [feature request](https://github.com/o3de/o3de.org/issues/new/choose) or start a discussion in [Discord](https://discord.gg/p3padwr58u) channel SIG-Testing!
+{{< /note >}}
+
+### Registering a new Python test
+
+Registering a Python-based test is simpler than registering a C++ test. However, it still requires that you define the C++ library that you want to test, which is likely already completed before designing integration tests. The steps below assume the production library is already defined. You can read more about defining production code in [Getting Started with the CMake Build System](/docs/user-guide/build/).
+
+#### Step 1: Register a PyTest target
+
+Find the `CMakeLists.txt` that defines the system you are testing. The file should exist at a path similar to `o3de/.../<MyModule>/CMakeLists.txt`. Depending on the test, it may be more appropriate to register the test to a `CMakeLists.txt` file at another location. Reasons include:
+
+* A test for a sub-module should exist at a child directory of its module.
+* A broad test integrating across multiple features should exist in a parent directory of the features.
+* A test which relies on code or assets in a game project must exist in that game project.
+  * This avoids the case where disabling the project breaks the still-registered tests.
+
+When you find the right `CMakeLists.txt` to register the test, add a line similar to the following:
+
+```
+ly_add_pytest(
+    NAME myPythonTest 
+    PATH ${CMAKE_CURRENT_LIST_DIR}/Tests/test_MyFile.py
+    TEST_SERIAL # many larger-than-unit tests can interfere with one another in parallel
+    RUNTIME_DEPENDENCIES # helps the test build-and-run from an IDE
+        Legacy::Editor
+        AZ::AssetProcessor
+        AutomatedTesting.Assets
+)
+```
+
+To verify everything is set up correctly, run the [CMake configure command](/docs/user-guide/build/configure-and-build/) from the CMake CLI or CMake GUI. This registers everything you just added and emits errors if anything was misconfigured.
```

Comment:
Similar to previous comment, can you specify the link to "CMake configure command", as there are numerous command samples in the linked **Configure and Build** page? Also, if its a single command, consider writing the command here, so users don't need to redirect to another page to find the command. You can keep the link to the "CMake configure command" page so users can find the thorough instructions there, and for the steps on how to do it via CMake GUI.

Reply:
Similarly, I do not want to describe the content from another section.

Reply:
Similar to response above, I suggest at least making the link text more accurate to reflect what the user will actually find when they click on it.

```suggestion
To verify everything is set up correctly, run the CMake configure command from **CMake CLI** or **CMake GUI** (refer to the [Configure and Build](/docs/user-guide/build/configure-and-build/) section). This registers everything you just added, and emits errors if anything is misconfigured.
```